lib/screens/v2/candidate_generator/widgets/subway_status.ex

```diff
@@ -53,11 +53,11 @@ defmodule Screens.V2.CandidateGenerator.Widgets.SubwayStatus do
           []
       end
 
-    %{
+    %SubwayStatus.SubwayStatusAlert{
       alert: alert,
       context: %{all_platforms_at_informed_station: all_platforms_at_informed_station}
     }
   end
 
-  defp append_context(alert, _), do: %{alert: alert, context: %{}}
+  defp append_context(alert, _), do: struct(SubwayStatus.SubwayStatusAlert, alert: alert)
```

Comment:
Since the struct is known here we can say `%SubwayStatus.SubwayStatusAlert{alert: alert}` and get a bit more compile-time verification.

Reply:
Woops, I had the commit to change that locally but forgot to push it up 😬